src/api.js

```diff
@@ -16,8 +16,6 @@ import * as time from 'lib0/time'
 const logWorker = logging.createModuleLogger('@y/redis/api/worker')
 // const logApi = logging.createModuleLogger('@y/redis/api')
 
-export const redisUrl = env.ensureConf('ysr-redis')
-
```

Comment:
Removing this export will break tsc:

```
> @hackmd/y-socketio-redis@0.0.5 lint
> standard && tsc

tests/api.tests.js:17:53 - error TS2339: Property 'redisUrl' does not exist on type 'typeof import("/home/stanley/Workspace/HackMD/y-socketio-redis/src/api", { with: { "resolution-mode": "import" } })'.

17   const redisClient = redis.createClient({ url: api.redisUrl })
                                                       ~~~~~~~~

tests/socketio.tests.js:69:53 - error TS2339: Property 'redisUrl' does not exist on type 'typeof import("/home/stanley/Workspace/HackMD/y-socketio-redis/src/api", { with: { "resolution-mode": "import" } })'.

69   const redisClient = redis.createClient({ url: api.redisUrl })
                                                       ~~~~~~~~
```

Maybe we can just change these tests to be:
```js
const redisClient = redis.createClient({ url: env.ensureConf('ysr-redis') })
```

Reply:
Fixed! thanks for the reminder.